apps/website/src/pages/courses/[courseSlug]/index.tsx

```diff
@@ -21,10 +21,6 @@ import { CourseUnitsSection } from '../../../components/courses/CourseUnitsSecti
 const CoursePage = () => {
   const { query: { courseSlug } } = useRouter();
 
-  if (typeof courseSlug !== 'string') {
-    return 'Invalid course slug';
```

Comment:
I think I'd have a slight preference for rendering ProgressDots - otherwise we are going ahead through the rest of the component with a weird looking courseSlug (that I think might be `undefined`?).

But this also works